ush/r2d2/example_store_bkg.py

```diff
@@ -0,0 +1,40 @@
+import os
+from solo.configuration import Configuration
+import ufsda.r2d2
+import yaml
+
+
+def store_bkg(yamlfile):
+    config = Configuration(yamlfile)
+    ufsda.r2d2.store(config)
+
+
+if __name__ == "__main__":
+    # first create dict of config for UFS RESTART files
+    config = {
+        'start': '2021-12-21T00:00:00Z',
+        'end': '2021-12-21T18:00:00Z',
+        'step': 'PT6H',
+        'forecast_steps': ['PT6H'],
+        'source_dir': '/work/noaa/stmp/rtreadon/comrot/prufsda/',
```

Comment:
This will break when I eventually remove this directory.

Reply:
Correct. By changing this directory path to `$ROTDIR`, and changing `RESTART_GES` to `RESTART`, this should work for any global-workflow experiment though.

Reply:
OK.  As the filename explains, this is an example.